### PR TITLE
feat: Don't block UI on pending spaces

### PR DESCRIPTION
**Pull request by dmaretskyi**

### CODE REVIEW

File: packages/apps/composer-app/src/layouts/DocumentLayout/DocumentLayout.tsx [hunk 1]
```diff
@@ -19,8 +19,8 @@ export const DocumentLayout = () => {
   const shadow = useButtonShadow('base');
 
   const { spaceKey } = useParams();
-  const spaces = useSpaces();
-  const space = spaces.find((space) => abbreviateKey(space.key) === spaceKey);
+  const spaces = useSpaces({ all: true });
```
Comment (zhenyasav) [inline packages/apps/composer-app/src/layouts/DocumentLayout/DocumentLayout.tsx:22]:
```suggestion
  const spaces = useSpaces({ includePartialSpaces: true });
```

Comment (zhenyasav) [reply packages/apps/composer-app/src/layouts/DocumentLayout/DocumentLayout.tsx:22]: or something more intuitive than `all`

File: packages/core/mesh/messaging/src/messenger.test.ts [hunk 2]
```diff
@@ -313,4 +317,39 @@ describe('Messenger', () => {
       expect(count).toEqual(1);
     });
   });
+
+  describe.skip('load', () => {
```
Comment (zhenyasav) [inline packages/core/mesh/messaging/src/messenger.test.ts:321]: skip?

File: packages/core/mesh/messaging/src/messenger.test.ts [hunk 3]
```diff
@@ -313,4 +317,39 @@ describe('Messenger', () => {
...
+    test('many connections to KUBE', async () => {
```
Comment (zhenyasav) [inline packages/core/mesh/messaging/src/messenger.test.ts:322]: seems unrelated?

**Review by zhenyasav — CHANGES_REQUESTED** [review]

File: packages/apps/composer-app/src/layouts/DocumentLayout/DocumentLayout.tsx (hunk 1 above)
Comment (dmaretskyi) [reply packages/apps/composer-app/src/layouts/DocumentLayout/DocumentLayout.tsx:22]: 👍

File: packages/core/mesh/messaging/src/messenger.test.ts (hunk 2 above)
Comment (dmaretskyi) [reply packages/core/mesh/messaging/src/messenger.test.ts:321]: just wanted to keep the code here in case we need load-test signal

Comment (dmaretskyi) [reply packages/core/mesh/messaging/src/messenger.test.ts:321]: not really a test that should be running
